### PR TITLE
Adding long press section to MenuTrigger docs

### PR DESCRIPTION
**Pull request by LFDanLu**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 🧢 Your Project:
RSP

### CODE REVIEW

File: packages/@react-spectrum/menu/docs/MenuTrigger.mdx
Comment (reidbarber) [inline packages/@react-spectrum/menu/docs/MenuTrigger.mdx:81]:
```suggestion
    <Item textValue="Crop Rotate">
```

**Review by reidbarber — COMMENTED** [review]
Looks great, just one small thing

File: packages/@react-spectrum/menu/docs/MenuTrigger.mdx
Comment (snowystinger) [inline packages/@react-spectrum/menu/docs/MenuTrigger.mdx:61]:
```suggestion
By default, a MenuTrigger's Menu is opened by pressing the trigger element or activating it via the <Keyboard>Space</Keyboard> or <Keyboard>Enter</Keyboard> keys. However, there may be cases in which your trigger element
```

File: packages/@react-spectrum/menu/docs/MenuTrigger.mdx
Comment (snowystinger) [inline packages/@react-spectrum/menu/docs/MenuTrigger.mdx:62]:
```suggestion
should perform a separate default action, such as selection, on press and should only display the Menu when long pressed. This behavior can be set by providing "longPress" to the `trigger` prop. Now the Menu will only be opened upon
```

File: packages/@react-spectrum/menu/docs/MenuTrigger.mdx
Comment (snowystinger) [inline packages/@react-spectrum/menu/docs/MenuTrigger.mdx:67]: Maybe a better example would be one that swaps out the tool for whatever is selected? Essentially a demo of exactly how photoshop uses this component?

File: packages/@react-spectrum/menu/docs/MenuTrigger.mdx
Comment (snowystinger) [inline packages/@react-spectrum/menu/docs/MenuTrigger.mdx:74]: now that i see it in action, technically we should apply selection like we do to action groups to the action button trigger when clicked and unset it when clicked again, it's a toggle
I don't think we support that on ActionButton's though, so... maybe your original example would be less confusing

don't know what do you think?

Comment (snowystinger) [reply packages/@react-spectrum/menu/docs/MenuTrigger.mdx:74]: In actiongroup it looks like we apply a classname for is-selected

Comment (LFDanLu) [reply packages/@react-spectrum/menu/docs/MenuTrigger.mdx:74]: yeah, thats a good point, I'll revert it to the original example. Really this example is just to show the functionality of longPress so keeping it simple is more beneficial IMO

**Review by snowystinger — DISMISSED** [review]

**Review by reidbarber — DISMISSED** [review]

**Comment by snowystinger** [comment]
Waiting for release before merging

File: packages/@react-spectrum/menu/docs/MenuTrigger.mdx
Comment (devongovett) [inline packages/@react-spectrum/menu/docs/MenuTrigger.mdx:96]: This makes the whole events section into a code block

File: packages/@react-spectrum/menu/docs/MenuTrigger.mdx
Comment (devongovett) [inline packages/@react-spectrum/menu/docs/MenuTrigger.mdx:62]:
```suggestion
should perform a separate default action on press such as selection, and should only display the Menu when long pressed. This behavior can be changed by providing `"longPress"` to the `trigger` prop. With this prop, the Menu will only be opened upon
```

File: packages/@react-spectrum/menu/docs/MenuTrigger.mdx
Comment (devongovett) [inline packages/@react-spectrum/menu/docs/MenuTrigger.mdx:60]: I think events should come before this section

**Review by devongovett — CHANGES_REQUESTED** [review]

File: packages/@react-spectrum/menu/docs/MenuTrigger.mdx
Comment (LFDanLu) [reply packages/@react-spectrum/menu/docs/MenuTrigger.mdx]: 🤦, mea culpa

File: packages/@react-spectrum/menu/docs/MenuTrigger.mdx
Comment (ktabors) [inline packages/@react-spectrum/menu/docs/MenuTrigger.mdx:89]: I noticed that `option + up arrow` also triggers a long press, but the focus is on the last element in the menu.

**Review by ktabors — APPROVED** [review]
LGTM

File: packages/@react-spectrum/menu/docs/MenuTrigger.mdx
Comment (snowystinger) [reply packages/@react-spectrum/menu/docs/MenuTrigger.mdx]: yep, i think we agreed it was overkill to explain that when the docs already mention the focus strategy

**Review by devongovett — APPROVED** [review]

**Comment by dannify** [comment]
To be merged after release

**Review by dannify — APPROVED** [review]

**Review by devongovett — APPROVED** [review]

**Pull request merged**
